deploy/0075_deploy_gameToken.ts

```diff
@@ -0,0 +1,26 @@
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {DeployFunction} from 'hardhat-deploy/types';
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const {deployments, getNamedAccounts} = hre;
+  const {deploy} = deployments;
+  const {deployer, gameTokenAdmin} = await getNamedAccounts();
+  const metaTransactionContract = await deployments.get(
+    'NativeMetaTransactionProcessor'
```

Comment:
The SAND contract is the `MetaTransactionProcessor`
the `NativeMetaTransactionProcessor` is actually working through SAND

Reply:
got it.